mkdocs/docs/HPC/jupyter.md

```diff
@@ -0,0 +1,129 @@
+# Jupyter notebook
+
+## What is a Jupyter notebook
+
+A [Jupyter notebook](https://jupyter.org/) is an interactive, web-based environment that allows you to create documents that contain live code, equations, visualizations, and plaintext. The code blocks in these documents can be used to write Python, Java, R and Julia code, among others. The combination of code executions with text and visual outputs make it a useful tool for data analysis, machine learning and educational purposes.
+
+## Using Jupyter Notebooks on the HPC
+
+### Starting a notebook using the web portal
+
+Through the web portal you can easily start a Jupyter notebook on a workernode, via the *Jupyter Notebook* button under the *Interactive Apps* menu item.
+
+<center>
+![image](img/ood_start_jupyter.png)
+</center>
+
+### Importing libraries
```

Comment:
@EwDa291 Please restructure this a bit, first focus on the basic use case of creating a Jupyter notebook session without needing anything extra.

Then add another section towards the end entitled `### Using extra Python packages`